ports.mk

```diff
@@ -92,11 +92,139 @@ endif
 	cp $(srcdir)/dists/iphone/icon-72.png $(bundle_name)/
 	cp $(srcdir)/dists/iphone/Default.png $(bundle_name)/
 
+ios7bundle: ios7
+	mkdir -p $(bundle_name)
+	awk 'BEGIN {s=0}\
```

Comment:
What's the reason we do this here, instead of keeping it in the `Info.plist` file already?